pages/api/feedContent/[id].ts

```diff
@@ -0,0 +1,146 @@
+import type {
+  FeedComment,
+  FeedItemContent,
+} from "../../../components/feed/FeedPage";
+
+const api_url =
+  "https://firestore.googleapis.com/v1/projects/dkms-spotify/databases/(default)/documents/feed_content";
+const base_url = "https://firestore.googleapis.com/v1/";
+
+type FeedCommentDocument = {
+  name: string;
+  fields: {
+    username: {
+      stringValue: string;
+    };
+    comment: {
+      stringValue: string;
+    };
+  };
+  createTime: Date;
+};
+
+type FeedCommentResponse = {
+  documents: FeedCommentDocument[];
+};
+
+type FeedContentDocument = {
+  name: string;
+  fields: {
+    username: { stringValue: string };
+    content: { stringValue: string };
+  };
+  createTime: Date;
+};
+
+type FeedContentResponse = {
+  documents: FeedContentDocument[];
+};
+
+function uniqueId() {
+  const dateString = Date.now().toString(36);
+  const randomness = Math.random().toString(36).substr(2);
+  return dateString + randomness;
+}
+
+export async function postFeedComment(
+  docId: string,
+  username: string,
+  comment: string
+) {
+  const comment_id = uniqueId();
+  const url = base_url.concat(docId, "/feed_comments/", comment_id);
+  await fetch(url, {
+    method: "PATCH",
+    headers: { "Content-type": "application/json" },
+    body: JSON.stringify({
+      fields: {
+        username: {
+          stringValue: username,
+        },
+        comment: {
+          stringValue: comment,
+        },
+      },
+    }),
+  });
+}
+
+async function getFeedComments(docId: string) {
```

Comment:
```suggestion
const getFeedComments = async (docId: string) => {
```

Reply:
just do this for all the functions so I dont have to comment on each one lol

I thought ESLint auto-converted to arrow functions but I guess not